README.md

```diff
@@ -1 +1 @@
-Be cool
+I like cooking, especially Chinese and Japanese food.
```

Comment:
can you see the picture？
![QQ_1725102121815](https://github.com/user-attachments/assets/3001250c-387f-4c54-9303-c8f33033fc01)